modules/funfact.py

```diff
@@ -12,4 +13,7 @@ def fill_queue(self):
     def response(self, query, message):
         if len(self.facts) == 0:
             self.fill_queue()
-        return self.facts.pop()
+        raw_text = self.facts.pop()
+        cleanr = re.compile('<.*?>')
```

Comment:
If you're bothering to compile the regular expression, it would be better to store it long-term, i.e. make it a class variable or create it in the __init__ function. Additionally, you might be able to save some resources by checking if the text contains `'<'` or `'>'` before running the regex replace. There are not very many facts that have HTML tags.

Actually, now that I think of it, you might get better results by simply replacing `<em>` and `</em>` with nothing. Those are the only HTML tags I've seen.